<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brisaConfGuiClass</class>
 <widget class="QMainWindow" name="brisaConfGuiClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Brisa Configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>481</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="expandButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Expand All Sections</string>
       </property>
       <property name="text">
        <string>Expand</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../usr/share/pixmaps/plusc2.png</normaloff>../../../../../../usr/share/pixmaps/plusc2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collapseButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Collapse All Sections</string>
       </property>
       <property name="text">
        <string>Collapse</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../usr/share/pixmaps/minusc2.png</normaloff>../../../../../../usr/share/pixmaps/minusc2.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>483</width>
      <height>338</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>301</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>481</width>
         <height>301</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Section</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>192</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label1Session">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Session</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSection">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label1Parameter">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Parameter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditParameter">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelValue">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditValue">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>701</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ConfigPath">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Configuration file path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConfigPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChange">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBoxSaveDefault">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Save on default configuration path (need to be root)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditSection</tabstop>
  <tabstop>lineEditParameter</tabstop>
  <tabstop>lineEditValue</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>collapseButton</tabstop>
  <tabstop>pushButtonRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
